observability/logging/logging-6.1/log6x-clf-6.1.adoc

```diff
@@ -62,9 +62,9 @@ Filters:: Transform or drop log messages in the pipeline. Users can define filte
 
 Inputs are configured in an array under `spec.inputs`. There are three built-in input types:
 
-application:: Selects logs from all application containers, excluding those in infrastructure namespaces such as `default`, `openshift`, or any namespace with the `kube-` or `openshift-` prefix.
+application:: Selects logs from all application containers, excluding those in infrastructure namespaces such as `default`, `kube`, `openshift`, or any namespace with the `kube-` or `openshift-` prefix.
 
-infrastructure:: Selects logs from infrastructure components running in `default` and `openshift` namespaces and node logs.
+infrastructure:: Selects logs from infrastructure components running in `default`, `kube`, and `openshift` namespaces and node logs.
```

Comment:
I think we should also mention
> or any namespace with the kube- or openshift- prefix.

as part of the infrastructure definition.

Reply:
done